widgets/pluginsCatalog/src/PluginsCatalogList.tsx

```diff
@@ -24,12 +19,16 @@ export interface PluginsCatalogItem extends Omit<PluginDescription, 'wagon_urls'
 
 const translate = Stage.Utils.getT('widgets.pluginsCatalog');
 
-function toUploadData(item: PluginsCatalogItem) {
+const getPluginKey = (plugin: PluginsCatalogItem) => {
+    return `${plugin.display_name.replaceAll(' ', '')}_${plugin.uploadedVersion}`.toLowerCase();
```

Comment:
We can make this concatenation more readable

```suggestion
    const pluginName = plugin.display_name.replaceAll(' ', '').toLowerCase();
    return `${pluginName}_${plugin.uploadedVersion}`;
```

Not sure if `toLowerCase()` needs to be casted also on `plugin.uploadedVersion`, so I've executed it only for the plugin name.

Let me know WDYT 👍